hack/update-generated-clientsets.sh

```diff
@@ -28,7 +28,7 @@ function generate_clientset_for() {
   grouppkg=$(realpath --canonicalize-missing --relative-to=$(pwd) ${package}/..)
   client-gen --clientset-path="${grouppkg}/generated" \
              --input-base="${package}"                            \
-             --output-base="../../.."                           \
+             --output-base="${GOPATH}/src"                            \
```

Comment:
The tool tries to write to `../../../github.com/openshift/origin/pkg/authorization/generated/internalclientset/doc.go`.

There is `_output/local/go/src/github.com/openshift` containing symlink `origin -> /home/test/openshift-origin`.

So the output path is `_output/local/go/src/github.com/openshift/origin/pkg/authorization/generated/internalclientset/doc.go`, a.k.a. `pkg/authorization/generated/internalclientset/doc.go`.